quixstreams/sinks/core/influxdb3.py

```
@@ -261,16 +269,19 @@ def write(self, batch: SinkBatch):
                     tags["__topic"] = batch.topic
                     tags["__partition"] = batch.partition
 
-                fields = (
-                    {
-                        field_key: value[field_key]
-                        for field_key in _fields_keys
-                        if (field_key in value or not self._allow_missing_fields)
-                        and field_key not in _tags_keys
```

This check was evicted. I'm not sure what was it for, just want to make sure it is an intended change 😅

---

Good eye! I removed it because it was a redundant check. 

Basically the goal is to make sure the same `key` is not in both `fields` and `tags` because it throws errors if you try to include it in both when pushing to InfluxDB.

There's a previous step that does:

```
tags = {}
for tag_key in _tags_keys:
    tag = value.pop(tag_key)
    tags[tag_key] = tag
```

Which means that for this:

```
...
if (field_key in value or not self._allow_missing_fields)
and field_key not in _tags_keys
...
```
 if a given key in `fields` was also in `tags`, it should have been popped from the `value` dict previously, which means the check `if (field_key in value...)` would return `False` as desired, and  `and field_key not in _tags_keys` would basically never resolve when it would theoretically matter and always be True otherwise.

On top of that, we prob shouldn't quietly ignore an issue like that anyway and instead let it raise, esp since we already raise for this problem when users pass static lists for `fields` and `keys` that overlap since it's an easy one time check.